src/avram/model.cr

```diff
@@ -147,6 +149,12 @@ abstract class Avram::Model
     )
   end
 
+  macro validate_columns
+    {% if COLUMNS.empty? %}
+      {% raise "#{@type} must define at least one column." %}
```

Comment:
Can we add the typical `Try this...` example here too? Like `column my_column : String` would be fine I think.

Reply:
<img width="440" alt="CleanShot 2021-07-12 at 18 16 50@2x" src="https://user-images.githubusercontent.com/17329408/125364981-762d6d00-e341-11eb-860d-431faeb1a539.png">

Updated it to match the [`validate_primary_key` error](https://github.com/luckyframework/avram/blob/98ed8b7924356a2b2999c5ef34db637d3856f8a3/src/avram/model.cr#L111-L125)